pyproject.toml

```diff
@@ -24,3 +24,14 @@ archs = 'x86_64 arm64'
 
 [tool.black]
 line-length = 92
+
+
+[tool.mypy]
+packages = ['sleepecg']
+exclude = '^sleepecg/test/'
+ignore_missing_imports = true
+disallow_untyped_defs = true
+no_implicit_optional = true
```

Comment:
Is this still necessary? According to the [docs](https://mypy.readthedocs.io/en/stable/config_file.html#confval-implicit_optional), the argument is now called `implicit_optional`, which is `false` by default. So I think we don't need this at all since the default is what we want.

Reply:
You're right!